### PR TITLE
Cherrypick 1198

### PR DESCRIPTION
**Pull request by david-martin**
## Additional Information
Cherrypick of commits from https://github.com/integr8ly/installation/pull/1198/commits

### CODE REVIEW

File: inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-integreatly_version: release-1.6.0-er1
+integreatly_version: master
```
Comment (wei-lee) [inline inventories/group_vars/all/manifest.yaml]: should this be changed?

File: playbooks/upgrades/upgrade.yaml [hunk 2]
```diff
@@ -265,6 +266,17 @@
...
+    - name: Update walkthroughs
+      include_role:
+        name: walkthroughs
+        tasks_from: upgrade
+    
+    # User SSO (evals realms for new walkthroughs)
+    - name: Create evalsNN realms in user sso for new walkthroughs
+      include_role:
+        name: rhsso-user
+        tasks_from: create-eval-user-realms
```
Comment (wei-lee) [inline playbooks/upgrades/upgrade.yaml]: will this run in all environments? I thought POC and OSD environments don't have evals users setup?

Comment (david-martin) [reply playbooks/upgrades/upgrade.yaml]: The number of eval users would be 0 in those environments.
@pmccarthy can you confirm, and as such this block wouldn't end up creating any realms in the user sso

Comment (pmccarthy) [reply playbooks/upgrades/upgrade.yaml]: @david-martin @wei-lee Correct, on OSD clusters eval user count is set to 0, see https://github.com/integr8ly/installation/blob/master/inventories/group_vars/osd/osd.yml#L34-L36

Hoping that the playbooks should handle that and move on

File: inventories/group_vars/all/manifest.yaml (hunk 1 above)
Comment (david-martin) [reply inventories/group_vars/all/manifest.yaml]: reverted

**Pull request merged**